shared/components/ember-output.jsx

```diff
@@ -0,0 +1,73 @@
+/* global React */
+
+export default React.createClass({
+    displayName: 'EmberOutput',
+
+    propTypes: {
+        source  : React.PropTypes.string.isRequired,
+        context : React.PropTypes.object.isRequired,
+        onChange: React.PropTypes.func
+    },
+
+    componentWillReceiveProps: function (nextProps) {
+        if (nextProps.source !== this.props.source) {
+            this.setState({});
+        }
+
+        Object.keys(nextProps).forEach(function (key) {
+            if (key === 'locales') {
+                Ember.set(this.service, 'locales', nextProps[key]);
+            }
+
+            this.controller.set(key, nextProps[key]);
+        }, this);
+    },
+
+    localesChanged: function () {
+        this.props.onChange(this.service.get('locales'));
+    },
+
+    componentWillUnmount: function () {
+        this.service.off('localesChanged', this, this.localesChanged);
+    },
+
+    componentDidMount: function () {
+        window.Ember.Application.initializer({
+            name : this.props.exampleId,
+            after: 'ember-intl-standalone',
+
+            initialize: function (container, app) {
+                var domElement = this.refs.placeholder.getDOMNode();
+                var controller = this.controller = container.lookupFactory('controller:basic').create();
+
+                this.service = container.lookup('intl:main');
+
+                if (!container.has('view:default')) {
+                    container.register('view:default', Ember.View);
+                }
+
+                this.service.on('localesChanged', this, this.localesChanged);
+
+                controller.setProperties(Object.assign({}, this.props.context, {
+                    locales  : this.props.locales,
+                    formats  : this.props.formats,
+                    messages : this.props.messages
```

Comment:
It's fine if these values are not set on the controller, but they should still be used as the source of truth for the overall state of the example and update the Ember app according. In React the props you get from your parent run the show. **Do we want to create an Ember app for each example?**